src/common/iop_profile.c

```diff
@@ -851,13 +851,29 @@ dt_ioppr_set_pipe_output_profile_info(struct dt_develop_t *dev,
   return profile_info;
 }
 
-dt_iop_order_iccprofile_info_t *dt_ioppr_get_histogram_profile_info(struct dt_develop_t *dev)
+static void _dt_ioppr_get_histogram_profile_type(dt_colorspaces_color_profile_type_t *profile_type,
+                                                 const char **profile_filename,
+                                                 const int force_normal_mode);
+
+static dt_iop_order_iccprofile_info_t *_dt_ioppr_get_histogram_profile_info(struct dt_develop_t *dev,
+                                                                            const int intent,
+                                                                            const int force_normal_mode)
```

Comment:
force_normal_mode looks like a boolean value, should be a gboolean

there is similar change to be done in other functions below.

Reply:
Indeed, thanks.